setup.py

```diff
@@ -134,7 +134,10 @@ def check_dependency_versions(min_versions):
                               (spversion, min_versions['scipy']))
 
     try:
-        from pandas.version import short_version as pversion
+        import pandas
+        #FIXME: this will break for pandas 1.0.0.  Needs elaborate parsing now,
+        # due to pandas removing version.short_version
```

Comment:
I'm pretty sure that that's broken. It's why there's a ``NumpyVersion`` class in scipy and statsmodels.